pallets/transaction-payment/src/lib.rs

```diff
@@ -234,8 +301,8 @@ where
 {
     /// Query the data that we know about the fee of a given `call`.
     ///
-    /// As this module is not and cannot be aware of the internals of a signed extension, It only
-    /// interprets the extrinsic as some encoded value and accounts for its weight
+    /// This module is not and cannot be aware of the internals of a signed extension, for example
+    /// a tip. It only interprets the extrinsic as some encoded value and accounts for its weight
```

Comment:
We remove the tip wording intentionally as we are not supporting the tip functionality in our chain.